update-tar-build/README.md

```diff
@@ -0,0 +1,12 @@
+# Update.tar builder
+
+## Guide
+
+- Download the deploy-files.tar.gz from the lamassu DO space
+- Extract it to a folder of the same name on the project root
+- Add a the fonts (also on DO space) to deploy-files
+- Build the Dockerfile by running the following command on project root:
```

Comment:
Can we combine this into a single script, possibly as part of the docker build? Consolidate the fonts into deploy-files.tar.gz so that we only have to manually download one file, then do the rest in the script.